packaging/dbscripts/upgrade/04_05_0180_mdevs_as_vmdevices.sql

```diff
@@ -0,0 +1,45 @@
+CREATE FUNCTION tmp_custom_properties_to_vm_devices() RETURNS void AS $$
```

Comment:
Did that was tested on scaled environment to see if we have to optimize it for env with many VMs ???

Reply:
No.  There can be two problems, I assume:

- The initial LIKE search for vm's with mdev_type.  I guess it shouldn't be a big issue even with thousands of VMs.

- The conversion.  I think environments with thousands of VMs having assigned vGPU mdev types are unlikely to exist.

I can attempt to make a quick test in my database to see if any of the above can be a real problem.

Reply:
As discussed offline with @ahadas and others: Indeed, environments with many VMs having the mdev_type property are unlikely.  And if I try the conversion on a couple of dozens VMs, it's instant in my database, so it's unlikely to be an issue needing or worth of optimization.